WORKSPACE

```diff
@@ -33,11 +37,10 @@ http_archive(
 )
 
 # Angular material
-# Note: material v7.1.1 is compatible with angular v7.1.0 under Bazel
 http_archive(
     name = "angular_material",
-    url = "https://github.com/angular/material2/archive/7.1.1.zip",
-    strip_prefix = "material2-7.1.1",
+    url = "https://github.com/gregmagolan/material2/archive/7.1.1-compat-ng-7.1.3.zip",
```

Comment:
comment about why? link to a PR?

Reply:
added comment. no PR on this one. when semver checks land in angular & rules_typescript then it will be possible to update the angular version in this repo without updating material. for now this is using a branch on my fork that allows the latest material release (7.1.1) to work with angular 7.1.3.